nncf/common/factory.py

```diff
@@ -25,3 +25,23 @@ def create(model):
             return GraphConverter.create_nncf_graph(model)
         raise RuntimeError('Cannot create backend-specific graph'
                            'because {} is not supported!'.format(model_backend))
+
+class ModelTransformerFactory:
```

Comment:
Could you please put some docstrings here and below? Don't forget to add typehints
```python
def create(model) -> ModelTransformer:
    ...
```
and
```python
def create(model) -> Engine:
    ...
```

Reply:
Yeah, missed it. Thanks.